test/unit_tests/aievec_tests/lit.local.cfg

```diff
@@ -0,0 +1,9 @@
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Copyright (C) 2024, Advanced Micro Devices, Inc. All rights reserved.
+
+if 'AIE' not in config.vitis_components:
```

Comment:
This is wrong, there are both aie1 and aie2 tests here.

Reply:
> This is wrong, there are both aie1 and aie2 tests here.

...and they use `xca_udm_dbg` which I think we can assume is present if `aiesimulator` is found.